github_discovery.py

```diff
@@ -187,6 +190,21 @@ def is_ipallowList_enabled(yaml_data):
         ip_allow_list_enabled = True
   return ip_allow_list_enabled
 
+def get_trivy_scan_json_data(project_name): 
+    
+  circleci_headers = {"Authorization": f"Circle-Token: {CIRCLECI_TOKEN}", "Content-Type": "application/json", "Accept": "application/json"}
+  project_url = f"{CIRCLECI_API_ENDPOINT}"+ project_name
+  try:                       
+      curl_command = f"""curl -H "Circle-Token: {CIRCLECI_TOKEN}" {project_url} | jq -r .[].build_num | head -n 1 | xargs -I {{}} echo "{project_url}/{{}}/artifacts" | xargs curl -s -H "Circle-Token: {CIRCLECI_TOKEN}" | jq -r .[].url | grep -E 'results.json' | head -n 1"""
```

Comment:
We should use the `requests` library - dont need subprocess either.

Reply:
Updated the code